test/general-tests/predeployed-accounts-test/check.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 set -e
 
+lsof -i :5050
```

Comment:
Why do we need this?

Reply:
Accidentally left this debugging line, removed it here: https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/253/commits/a836aa58a35ca9e79550838f83eb953df3644e5a